praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def archive(self):
+        """Archive the conversation."""
```

Comment:
I'm pretty sure I haven't done that on any of the methods so far, but I'll go back and add them soon.